### PR TITLE
Improve query method & Connection class

### PR DESCRIPTION
**Pull request by marcelluzs**

### CODE REVIEW

**Comment by marcelluzs** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/marcelluzs/pypostgres/pull-request/4/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=marcelluzs/pypostgres)

**Comment by ryukinix** [comment]
Let's wait for the build progress to avoid pain.

File: pypostgres/pypostgres.py [hunk 1]
```diff
@@ -40,9 +47,11 @@ def build_dataframe(result_set, columns):
             df.loc[index] = items
         return df
 
-    def to_dataframe(self, table, columns='*', conditions=None):
+    def select_to_df(self, table, columns='*', conditions=None):
```
Comment (ryukinix) [inline pypostgres/pypostgres.py]: Nice method name

**Comment by ryukinix** [comment]
:+1:

**Pull request merged**